Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.11.3
+FROM node:8.12.0
```

Comment:
I'm surprised this is the only place we need to change this version! But looks like we're not pinning anything in the `package.json`

Reply:
yes `package.json` seems to only pin to `8.x.x`